pkg/ingester/metrics.go

```diff
@@ -339,6 +349,12 @@ type tsdbMetrics struct {
 	tsdbChunksRemovedTotal       *prometheus.Desc
 	tsdbMmapChunkCorruptionTotal *prometheus.Desc
 
+	tsdbExemplarsTotal          *prometheus.Desc
```

Comment:
Could you add these new metrics to the existing test in `pkg/ingester/metrics_test.go`, please?